### PR TITLE
Fixed missing on total no filter

### PR DESCRIPTION
**Pull request by knil-maloon**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -261,7 +261,18 @@ function getTotalCol($question,$project_id,$question_1,$conditionDate,$topScoreM
         }
     }
 
-    $missingOverall += $missing_col;
+    $row_questions_1 = ProjectData::getRowQuestionsParticipantPerception();
+    $RecordSetOverallMissing = \REDCap::getData($project_id, 'array', $recordIds, null, null, null, false, false, false, "[".$question_1."] = ''".$conditionDate);
```
Comment (mmcev106) [inline functions.php:265]: Should this getData() call include a field list so that all data is not pulled from the DB?

**Review by mmcev106 — CHANGES_REQUESTED** [review]

File: functions.php (hunk 1 above)
Comment (knil-maloon) [reply functions.php]: Good point. I just changed it :)
I also found an unused variable

File: functions.php [hunk 2]
```diff
@@ -261,7 +261,19 @@ function getTotalCol($question,$project_id,$question_1,$conditionDate,$topScoreM
         }
     }
 
-    $missingOverall += $missing_col;
+    $row_questions_1 = ProjectData::getRowQuestionsParticipantPerception();
+    $data = implode(",",$row_questions_1);
+    $RecordSetOverallMissing = \REDCap::getData($project_id, 'array', $recordIds, array($data), null, null, false, false, false, "[".$question_1."] = ''".$conditionDate);
```
Comment (mmcev106) [inline functions.php:266]: I don't believe this works as expected, which means this change was not fully tested before committing.  Please fully test all change before committing going forward.  If you're not sure how to fully test something, please ask.

**Review by mmcev106 — CHANGES_REQUESTED** [review]

File: functions.php (hunk 2 above)
Comment (knil-maloon) [reply functions.php]: I made a test but I think it was too small to realize that the data was working but not giving the right numbers 🤦‍♀️
Now I'm getting the correct data. I made a bigger sized test.

Comment (knil-maloon) [reply functions.php]: Thank you for checking this! I'll take a note to use the bigger sample set as reference.

Comment (mmcev106) [reply functions.php]: In my experience smaller test cases that more closely target the specific code changes made (similar to unit tests) are actually more effective at verifying changes like this than bigger sample sets that may or may not actually exercise the code that was changed.  They'll also typically quicker.  I wrote a quick test for this line in under a minute!

**Review by mmcev106 — APPROVED** [review]

File: functions.php (hunk 2 above)
Comment (knil-maloon) [reply functions.php]: 👍

**Pull request merged**